Assignment/Program.cs

```diff
@@ -1,7 +1,36 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
-   }
-}
\ No newline at end of file
+﻿using System.Text;
+using static System.Console;
+
+OutputEncoding = new UnicodeEncoding ();
+Write ("\u250c");
+for (int i = 0; i < 7; i++) Write ("\u2500\u2500\u2500\u252c");
+Write ("\u2500\u2500\u2500\u2510");
+WriteLine ();
+WriteLine ("\u2502 \u2656 \u2502 \u2658 \u2502 \u2657 \u2502 \u2654 \u2502 \u2655 \u2502 \u2657 \u2502 \u2658 \u2502 \u2656 \u2502");
+Write ("\u251C");
+for (int j = 0; j < 7; j++) Write ("\u2500\u2500\u2500\u253C");
+Write ("\u2500\u2500\u2500\u2524");
+WriteLine ();
+WriteLine ("\u2502 \u265F \u2502 \u265F \u2502 \u265F \u2502 \u265F \u2502 \u265F \u2502 \u265F \u2502 \u265F \u2502 \u265F \u2502");
+
+for (int k = 0; k < 4; k++) {

```

Comment:
Can you can rewrite the program with a for loop to render the board and based on the index you can select the chess board items to be printed.